TrafficCapture/trafficReplayer/src/test/java/org/opensearch/migrations/replay/TransformationLoaderTest.java

```diff
@@ -28,4 +28,28 @@ public void testTransformationLoader() throws Exception {
         Assertions.assertNotEquals(origDocStr, docWithNewHostnameStr);
     }
 
+    @Test
+    public void testThatSimpleNoopTransformerLoads() throws Exception {
+        var noopTransformer = new TransformationLoader()
+                .getTransformerFactoryLoader("localhost", "NoopTransformerProvider");
```

Comment:
Do we currently have any other existing transformation providers besides "NoopTransformerProvider" ?

Reply:
In the base package, no.  In TrafficReplayer, there's a HostTransformer & soon a UserAgentTransformer.  Transformers exist for Jolt and JMESPath in separate packages.